CMakeLists.txt

```diff
@@ -93,19 +93,21 @@ install(EXPORT umpire-targets DESTINATION share/umpire/cmake)
 add_subdirectory(src)
 
 if (ENABLE_TESTS)
-  set (UMPIRE_TEST_OUTPUT_DIR ${CMAKE_BINARY_DIR}/test)
+  set(UMPIRE_TESTS_OUTPUT_DIR ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/tests)
```

Comment:
I'm assuming `CMAKE_RUNTIME_OUTPUT_DIRECTORY` will be `bin`?

Reply:
Yeah, it's another variable defined by BLT